src/component/__tests__/Board.test.tsx

```diff
@@ -24,6 +25,8 @@ describe("Board", () => {
     endGame,
     finishTime: 0,
     setFinishTime,
+    checkLetter,
+    incorrectLetters: 0,
```

Comment:
I know these were added erroneously, but you are right, these should be in their own commit.

Reply:
@swcraig yeah and something I did with this caused a merge conflict with master so I'll have to deal with the repercussions of sneaking changes in :)